bugbear.py

```diff
@@ -685,6 +687,20 @@ def check_for_b018(self, node):
             ):
                 self.errors.append(B018(subnode.lineno, subnode.col_offset))
 
+    def check_for_b021(self, node):
+        if (
+            node.body
+            and isinstance(node.body[0], ast.Expr)
+            and isinstance(node.body[0].value, ast.JoinedStr)
+            and ast.get_source_segment(
+                "".join(self.lines),
+                node.body[0].value,
+            ).startswith('f"""')
```

Comment:
Thanks will add a couple extra unit tests for these as well! 😄

Reply:
Actually on second thought, since these are also possible we can just remove this condition altogether since `.startswith(('f"', "f'"))` is already covered by virtue of it being a joined str!

Reply:
Updated now 👍